packages/frontend/package.json

```diff
@@ -23,6 +23,7 @@
     "react-responsive": "^8.1.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "^3.4.3",
+    "react-select": "^3.1.0",
```

Comment:
Leave this comment to add some stress on you like "shit I have to change another thing!"
muahah